types/hedron/index.d.ts

```diff
@@ -1,172 +1,147 @@
+   * Height Property
+   */
+  height?: string;
+  /**
+   * Visibilty Property
+   */
+  visibility?: "visible" | "hidden" | "collapse" | "initial" | "inherit";
+  /**
+   * Display Property
+   */
+  display?: string;
+  /**
+   * Opacity Property
+   */
+  opacity?: number;
+  /**
+   * Background Property
+   */
+  background?: string;
+  /**
+   * Border property
+   */
+  border?: string;
+  /**
+   * Controls hidden
+   */
+  hidden?: boolean;
 }
 
-export interface PageProps {
-    /**
-     * You can specify component to use specific tag
-     * @default 'div'
-     */
-    tagName?: keyof ElementTagNameMap;
-    /**
-     * Draws all child columns with "bounding boxes" for easy
-     * visualization of the grid. This enables debug mode for all the
-     * children of this component
-     * @default false
-     */
-    debug?: boolean;
-    /**
-     * Enabling fluid mode disables the fixed width of the Page
-     */
-    fluid?: boolean;
-    /**
-     * If not using fluid, set a custom width for the page.
-     */
-    width?: string;
+export interface ProviderProps {
+  /**
+   * The amount of padding added
+   */
+  padding?: string;
+  /**
+   * Object specifying the breakpoints
+   * { mobile: "-500", tablet: "501-750", wide: "+750" }
+   */
+  breakpoints?: any;
 }
 
-export interface RowProps {
-    /**
-     * Draws all child columns with "bounding boxes" for easy
-     * visualization of the grid.
-     * @default false
-     */
-    debug?: boolean;
-    /**
-     * You can specify component to use specific tag
-     * @default 'section'
-     */
-    tagName?: keyof ElementTagNameMap;
-    /**
-     * The amount of horizontal columns this row creates.
-     * @default 12
-     */
-    divisions?: number;
-    /**
-     * Sets the value of the CSS align-content property
-     */
-    alignContent?: string;
-    /**
-     * Sets the value of the CSS align-items property
-     */
-    alignItems?: string;
-    /**
-     * Sets the value of the CSS align-self property
-     */
-    alignSelf?: string;
-    /**
-     * Sets the value of the CSS justify-content property
-     */
-    justifyContent?: string;
-    /**
-     * Sets the value of the CSS order property
-     */
-    order?: string;
+export interface BoundsProps extends Styleable {
+  /**
+   * Draws all child columns with "bounding boxes" for easy
+   * visualization of the grid. This enables debug mode for all the
+   * children of this component
+   * @default false
+   */
+  debug?: boolean;
+  /**
+   * Controls the CSS flex property
+   */
+  flex?: string;
+  /**
+   * Direction of content
+   */
+  direction?: "horizontal" | "vertical";
+  /**
+   * Alignment of children along the vertical axis
+   */
+  valign?: "top" | "center" | "bottom";
+  /**
+   * Alignment of children along the horizontal axis
+   */
+  halign?: "left" | "center" | "right";
+  /**
+   * Allow for customer props due to ability to add
+   * custom breakpoints
+   */
+  [x: string]: any;
```

Comment:
Can `Bounds` have "custom breakpoints" props? I didn't see it in the [docs](https://github.com/garetmckinley/hedron#props-1)